setup/install/providers/aws/aws-eks.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: single
+title:  "Amazon EKS"
```

Comment:
I don't think this should be listed as a separate cloud provider. These installation instructions should be under "kubernetes" rather than "aws".

Reply:
What if we created a new Kubernetes section but then added a link under "Amazon Web Services" similar to how it is now but links to the new section? I can imagine as more companies contribute, navigating might get a bit obtuse. My assumption is that a user will look for their platform, then look for the specific cloud provider. Thoughts?

Reply:
What do you mean by a new section? A new page with a subheader, or a new section on the Kubernetes page? 

I totally agree with having EKS specific docs under the Kubernetes page, just not with labeling it as a "provider" as this confuses things down the road.

Reply:
@pauljrob / @lwander , something like below works ?

<img width="184" alt="screen shot 2018-07-08 at 7 08 15 pm" src="https://user-images.githubusercontent.com/20847838/42427300-8fb393c0-82e2-11e8-9233-34ff350681aa.png">

Reply:
I think this is good, as long as (as @lwander has said) this is framed as: some things to do that are specific to EKS for setting up a k8s provider to deploy to, rather than to install Spinnaker on.

Reply:
done. thanks